blssig/aggregation.go

```diff
@@ -0,0 +1,77 @@
+package blssig
+
+import (
+	"fmt"
+
+	"github.com/filecoin-project/go-f3/f3"
+
+	"github.com/drand/kyber"
+	"github.com/drand/kyber/sign"
+	"github.com/drand/kyber/sign/bdn"
+	"golang.org/x/xerrors"
+)
+
+func (v Verifier) pubkeysToMask(pubkeys []f3.PubKey) (*sign.Mask, error) {
+	kPubkeys := make([]kyber.Point, 0, len(pubkeys))
+	for i, p := range pubkeys {
+		point := v.keyGroup.Point()
+		err := point.UnmarshalBinary(p)
+		if err != nil {
+			return nil, xerrors.Errorf("unarshalling pubkey at index %d: %w", i, err)
+		}
+		if point.Equal(v.keyGroup.Point().Null()) {
+			return nil, xerrors.Errorf("the public key at %d is a null point", i)
+		}
```

Comment:
I'd consider making a helper function for this.

Reply:
I considered it, decided that 2x reuse does not yet warrant it.

Reply:
I'm just concerned that someone will do this then _not_ check if it's null. But we're probably fine.